src/main/java/com/xebialabs/overthere/util/WindowsCommandLineArgsSanitizer.java

```diff
@@ -24,9 +24,18 @@
 
 public class WindowsCommandLineArgsSanitizer {
 
-    private static final char[] SPECIAL_CHARS = "|<>&^\r\n".toCharArray();
+    private static final char[] SPECIAL_CHARS = " |<>&^\r\n".toCharArray();
```

Comment:
Why is the space added to the SPECIAL_CHARS? The space should not be escaped with a caret, right?